RadixWallet/Prelude/Extensions/IdentifiedArrayOf+Extensions.swift

```diff
@@ -6,6 +6,22 @@ extension IdentifiedArrayOf {
 	}
 }
 
+// MARK: - ForEachStatic
+/// A wrapper around ForEach that can be used with **static** collections of elements that don't conform to `Identifiable`
+public struct ForEachStatic<Data: RandomAccessCollection, Content: View>: View {
```

Comment:
I'd prefer it if we did not use `Data` as generic type name. Maybe `Elements` ? which will read out:
```swift
public let elements: [OffsetIdentified<Elements.Element>]
```